libmpv/opengl_cb.h

```diff
@@ -163,9 +175,13 @@ extern "C" {
  *
  * glMPGetNativeDisplay("wl") should return a Wayland "struct wl_display *".
  *
- * glMPGetNativeDisplay("drm") should return a DRM FD casted to intptr_t (note
- * that a 0 FD is not supported - if this can happen in your case, you must
- * dup2() it to a non-0 FD).
+ * glMPGetNativeDisplay("drm") should return an array of void * containing
+ * 3 elements which are:
+ * 0 : pointer to the DRM fd (int). Note that a 0 FD is not supported - if this
+ *     can happen in your case, you must dup2() it to a non-0 FD).
+ * 1 : pointer to the crtc_id (int). This is the currently used crtc.
+ * 2 : pointer to the drmAtomicRequest that is being used for the renderloop.
+ *     This atomic request pointer should be usually created at every renderloop.
```

Comment:
Should be renamed since the ABI changes. Also maybe it's time to make this a struct. Make sure the header builds on any OS (i.e. without DRM headers), which might require the use of opaque structs or even just `void*`.

Reply:
Agreed, `void *` should probably be fine as long as we document that properly in the headers.